web/src/app/contact-methods/components/ContactMethodForm.js

```diff
@@ -191,23 +191,14 @@ class ContactMethodForm extends Component {
     this.createNotificationRule(this.state.contactMethod)
   }
 
-  renderVerificationForm() {
-    if (!this.state.showVerifyForm) return null
-    const { contactMethod, showVerifyForm } = this.state
-    return (
-      <VerificationForm
-        key='Verify code for Contact Method '
-        id={contactMethod.id}
-        name={contactMethod.name}
-        value={contactMethod.value}
-        type={contactMethod.type}
-        open={showVerifyForm}
-        userId={this.props.userId}
-        handleRequestClose={() => this.setState({ showVerifyForm: false })}
-        onSuccess={() => this.onVerificationSuccess()}
+  renderVerificationForm = () =>
+    this.state.showVerifyForm && (
+      <ContactMethodVerificationDialog
+        key='verify-dialog'
```

Comment:
Is key used?

Reply:
it's returned as an array in this (older) component, so it will throw an error without it